lib/matplotlib/cbook.py

```diff
@@ -492,17 +504,23 @@ def connect(self, s, func):
         func will be called
         """
         self._func_cid_map.setdefault(s, WeakKeyDictionary())
-        if func in self._func_cid_map[s]:
```

Comment:
This was not the ``dict`` you were thinking of?  A ``weak`` reference ;).  (Watching ``A New Hope`` as I type this, so the pun seems relevant).